nautobot/core/ui/object_detail.py

```diff
@@ -296,7 +296,7 @@ def __init__(
 
         Args:
             label (str): Label to display for this panel. Optional; if an empty string, the panel will have no label.
-            section (str): One of the `SectionChoices` values, indicating the layout section this Panel belongs to.
+            section (str): One of the [`SectionChoices`](./#nautobot.apps.ui.SectionChoices) values, indicating the layout section this Panel belongs to.
```

Comment:
These kind of links really make me wish we were using sphinx instead of mkdocstrings. Lots more automatic cross-linkage that way.

Reply:
I was thinking about that too... even looking for some plugin. But the only thing I've found is this: https://github.com/mkdocstrings/autorefs 

It won't resolve the problem but might simplify generating of links to just `[`SectionChoices`][nautobot.apps.ui.SectionChoices]`

Reply:
Looks interesting - maybe worth exploring today as a hackathon task?

Reply:
Will do that, but on Tuesday morning since we have Independence Day Holiday on Monday.